### ISSUE TITLE
[Chrome] Third step of the Premium Onboarding is missing content for users that have the Add-on installed

### ISSUE THREAD

**Issue by AOiegas** [mozilla/fx-private-relay#2189]
#### Build:
- Stage;
- Production;

#### Affected Platforms:
- Windows 10;

#### Browsers tested on:
- Chrome, Release version;

#### Prerequisites:
- Be signed in into a Free Firefox Relay account;
- Have the Relay add-on installed;

#### Steps to reproduce:
1. Go to the Relay dashboard;
2. Navigate to the purchase flow and complete a Premium subscription;
3. Navigate back to the Relay dashboard, where the Premium onboarding starts;
4. Click on the “Get Started” button on the first step;
5. Click on the “Continue, I’ll register my custom subdomain later” blue link on the second step;
6. Observe the third step of the onboarding process;

#### Expected result:
- The third step displays all the content, including a “Go to Dashboard” blue button, and a “Relay extension added!” notice;

#### Actual result:
- The third step is missing the “Go to Dashboard” button and the “Relay extension added!” notice;

#### Notes:
- Firefox doesn’t display this issue;
- This is reproducible on Stage as well;
- This happens for users that install the add-on during the third step as well;
- See attached media:
Chrome:
![woop2](https://user-images.githubusercontent.com/85748693/178513937-35b6c23a-99d8-46e2-9f1a-3ed7f4abafc1.png)

Firefox comparison:
![woop3](https://user-images.githubusercontent.com/85748693/178514019-585c99fd-44ad-410b-924b-987b1bdecbe1.png)

**Comment by AOiegas** [comment]
Yes, the user can get to the Dashboard if they click skip. It's not a dead-end, but there are big chances a user might be confused when presented only with the option to skip.

**Comment by groovecoder** [comment]
Agree - still a major bug. But we'll prioritize for next release rather than make an emergency release for it right now.
